binding.gyp

```diff
@@ -1,7 +1,7 @@
 {
   'targets': [
     {
-      'target_name': 'binding',
+      'target_name': 'bluetooth_hci_socket',
```

Comment:
Is it possible to use dashes in place of underscores here?

Reply:
No. We can use "<(module_name)" here but in package file it needs to be a valid C variable name, so no dashes.
See https://github.com/mapbox/node-pre-gyp#module_name